src/ui/CodeLensProvider.ts

```diff
@@ -77,6 +78,39 @@ export class CodelensProvider implements vscode.CodeLensProvider {
 					this.codeLenses.push(debugTestCodelens);
 				}
 			}
+
+			// The setting for the coverage code lens needs to be switched on for the mechanism
+			// to be enabled
+			if (vscode.workspace.getConfiguration('codelens-kani').get('highlightCoverage', true)) {
+				for (const harness of kani_harnesses) {
+					const harness_name = harness.harnessName;
+
+					if (harness_name === undefined || harness_name === '') {
+						return [];
+					}
+
+					const startPosition = harness.endPosition;
+
+					// This is the metadata that VSCode needs to place the codelens button
+					const line = document.lineAt(startPosition.row);
+					const regex = /fn\s+([\w_]+)/g;
```

Comment:
It's an optional variable that's used when VS Code is not able to find the position provided. But in this case, we can remove it safely, since we already provide the position.

Reply:
Removed, thanks for spotting.